sdk-commons/src/test/java/org/openmrs/maven/plugins/utility/VersionsHelperTest.java

```diff
@@ -18,63 +18,70 @@ public class VersionsHelperTest {
     VersionsHelper helper;
 
     @Before
-    public void before(){
+    public void before() {
         helper = new VersionsHelper(null, null, null, null);
     }
 
-    private List<ArtifactVersion> createTestVersions(String... versions){
+    private List<ArtifactVersion> createTestVersions(String... versions) {
         List<ArtifactVersion> list = new ArrayList<>();
-        for(String version: versions) {
+        for (String version : versions) {
             list.add(new DefaultArtifactVersion(version));
         }
         return list;
     }
 
     @Test
-    public void getLatestReleasedVersion_shouldNotIncludeSnapshots(){
+    public void getLatestReleasedVersion_shouldNotIncludeSnapshots() {
         String res = helper.getLatestReleasedVersion(
                 createTestVersions("1.8.8", "1.8.3", "1.8.10-SNAPSHOT", "1.8.9-alpha"));
         assertThat(res, is(equalTo("1.8.8")));
     }
+
     @Test
-    public void getLatestSnapshotVersion_shouldReturnLatestSnapshot(){
+    public void getLatestSnapshotVersion_shouldReturnLatestSnapshot() {
         String res = helper.getLatestSnapshotVersion(
                 createTestVersions("1.8.8", "1.8.3", "1.8.10-SNAPSHOT", "1.8.9-alpha"));
         assertThat(res, is(equalTo("1.8.10-SNAPSHOT")));
     }
+
     @Test
-    public void getSuggestedVersions_containSnapshots(){
+    public void getSuggestedVersions_containSnapshots() {
         List<String> res = helper.getSuggestedVersions(
                 createTestVersions("1.8.8", "1.8.3", "1.8.10-SNAPSHOT", "1.8.9-alpha"), 3);
         assertThat(res, hasItem("1.8.10-SNAPSHOT"));
     }
+
     @Test
-    public void getSuggestedVersions_containGivenNumberOfReleases(){
+    public void getSuggestedVersions_containGivenNumberOfReleases() {
         List<String> res = helper.getSuggestedVersions(
                 createTestVersions("1.8.8", "1.8.3", "1.8.10-SNAPSHOT", "1.8.9-alpha"), 1);
-        assertThat(res, hasItem("1.8.9-alpha"));
+        assertThat(res, hasItem("1.8.10-SNAPSHOT"));
```

Comment:
This seems like a bug in the test, which allowed bugs in the code to pass.  If asking for suggested versions with a max number of 1, it should return the latest suggested version, and 1.8.10 > 1.8.9